poetry.lock

```diff
@@ -1,4 +1,4 @@
-# This file is automatically @generated by Poetry 1.8.1 and should not be changed by hand.
```

Comment:
Is there a way to pin the version of poetry so we don't have this?

Reply:
I don't think you can - or not that I've found with a bit of Googling. You can't pin in `pyproject.toml` as poetry needs to be installed outside project environment (https://python-poetry.org/docs/#installing-with-pipx). This thread suggested you couldn't either (e.g. https://github.com/python-poetry/poetry/issues/3316).